src/Sql/dbo/Stored Procedures/Organization_ReadAbilities.sql

```diff
@@ -24,7 +24,7 @@ BEGIN
         [LimitCollectionCreation],
         [LimitCollectionDeletion],
         [AllowAdminAccessToAllCollectionItems],
-        [UseRiskInsights]
```

Comment:
We shouldn't remove `[UseRiskInsights]`, right?

Reply:
Good catch!

Reply:
I added it back here and the migration script